float64[3] tcpPmarker
float64[3] tcpRmarker
---
int8 ok